src/pages/ShikshalokamVoiceChat/dynamic-voice-chat.js

```
@@ -0,0 +1,2756 @@
+  useEffect(() => {
+    async function createUserProfile() {
+      try {
+        setIsLoading(true)
+
+        const response = await createUserProfileApi({
+          access_token: accessToken,
+        })
+
+        if (response) {
+          const data = response.profile_details
+          const preferredLanguage = preferredLanguage || {}
```

Biome (2.3.14)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)